### DISCUSSION TITLE
The version has been updated to the latest version 5.2.1, but after adding more than one text box, it will get stuck abnormally

### DISCUSSION THREAD

**Discussion by Charley-liu** [fabricjs/fabric.js#8150] [Q&A] [answered]
![20220816-102228](https://user-images.githubusercontent.com/23217036/184819102-756ef814-78c3-4544-81fd-b403cf6116bf.gif)
Because it's my first time to interact with the community, I don't quite understand. Sorry, I put forward two useless pr @ShaMan123

**Comment by ShaMan123** [comment, marked as answer]
Fill [this](https://github.com/fabricjs/fabric.js/issues/new?assignees=&labels=&template=bug_report.md) out. It is there for a reason. Do not take it for granted. No one can guess how you ended up with a bug.
You **MUST** provide a working precise (no junk or unrelated logic) reproduction. You can use codesandbox or jsfiddle (links are provided in the issue template link). Without that there is no way to understand what went wrong. Reproducing is the first step of any bug triaging.
If you stick to these instructions you will probably find that the bug is on your side.

**READ [CONTRIBUTING](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#issue-tracker)**

I do not wish to spend my time writing to people "please make an effort", "your bug report is unreadable"
